config.c

```diff
@@ -291,9 +347,8 @@ if (configPath[0] == '\0') {
 		configHome = getenv("HOME");
 		if (configHome != NULL) {
 			sprintf(configPath,"%s/%s/%s/", configHome, ".config", PACKAGE);
-		} else {
```

Comment:
I don't think you should have removed this line, now it will always fail with ERR_HOMELESS, unless you have XDG_CONFIG_HOME defined, wich some systems don't, like debian dabes systems.

Reply:
Ah, good catch. That was accidental, I'll add it back in.